app/models/card.rb

```diff
@@ -8,18 +8,43 @@ class Card < ActiveRecord::Base
   validate :clear_words, if: proc { |a| a.original_text? && a.translated_text? }
   validate :change_string, if: proc { |a| a.original_text? && a.translated_text? }
 
-  before_create :recheck_date
+  before_create :set_review_date_as_now
 
-  scope :ready_for_review, -> { where("review_date <= ?", Time.now) }
+  scope :ready_for_review, -> { where("review_date <= ?", Time.zone.now) }
   scope :random, -> { order('RANDOM()') }
   scope :all_cards, ->(packs) { where(pack_id: packs.pluck(:id)) }
 
   def check_word(user_text)
     original_text.casecmp(user_text.mb_chars.strip.downcase).zero?
   end
 
-  def recheck_date
-    self.review_date = (Date.today + 3.days).to_s
+  def set_review_date_as_now
+    self.review_date = Time.zone.now.to_s
+  end
+
+  def check_box_value
```

Comment:
so, i think it is not really good name for this method, smth like `set_review_date_and_box` will be better.